src/parser/syntax.ml

```diff
@@ -90,9 +90,9 @@ let rec inline_element
     fun input location next_token ->
 
   match next_token with
-  | `Space ->
+  | `Space ws ->
     junk input;
-    Location.at location `Space
+    Location.at location (`Space ws)
```

Comment:
This looks like a good place for an `as`-binding to avoid an allocation, which is introduced by this diff.